roles/neutron-data-network/files/etc/neutron/lbaas_templates/haproxy_base.j2

```diff
@@ -0,0 +1,34 @@
+{# # Copyright 2014 OpenStack Foundation
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+#}
+# Configuration for {{ loadbalancer_name }}
+global
+    daemon
+    user nobody
+    group {{ usergroup }}
+    log /dev/log local0
+    log /dev/log local1 notice
+    maxconn 200000
+    stats socket {{ sock_path }} mode 0666 level user
+
+defaults
+    log global
+    retries 3
+    option redispatch
+    timeout connect 5000
+    timeout client 50000
+    timeout server 50000
```

Comment:
These are lbaas supplied values -- the remember of these files other than the global maxconn setting is taken verbatim from neutron-lbaas.